sdk/cosmos/spark.yml

```diff
@@ -67,3 +67,18 @@ stages:
       DatabricksToken: $(spark-databricks-token)
       SparkVersion: 'azure-cosmos-spark_3-4_2-12'
       ClusterName: 'oltp-ci-spark34-2workers-ds3v2'
+  - template: /sdk/cosmos/spark.databricks.yml
+    parameters:
+      CosmosEndpoint: $(spark-databricks-cosmos-endpoint)
+      CosmosKey: $(spark-databricks-cosmos-key)
+      DatabricksEndpoint: $(spark-databricks-endpoint)
+      SubscriptionId: '8fba6d4f-7c37-4d13-9063-fd58ad2b86e2'
+      TenantId: '72f988bf-86f1-41af-91ab-2d7cd011db47'
+      ResourceGroupName: 'oltp-spark-ci'
+      ClientId: $(spark-databricks-cosmos-spn-clientId)
+      ClientSecret: $(spark-databricks-cosmos-spn-clientSecret)
+      CosmosContainerName: 'sampleContainer5'
+      CosmosDatabaseName: 'sampleDB5'
+      DatabricksToken: $(spark-databricks-token)
+      SparkVersion: 'azure-cosmos-spark_3-5_2-12'
+      ClusterName: 'oltp-ci-spark34-2workers-ds3v2'
```

Comment:
This needs to be the clusterName of a new Cluster with Spark 3.5

Reply:
Otherwise we have zero test coverage in actual Spark live test

Reply:
changed